src/Sylius/Behat/Context/Api/Admin/ManagingAdministratorsContext.php

```diff
@@ -129,13 +155,41 @@ public function iDeleteAdministratorWithEmail(AdminUserInterface $adminUser): vo
     }
 
     /**
+     * @Given /^I am editing (my) details$/
      * @When /^I want to edit (this administrator)$/
      */
     public function iWantToEditThisAdministrator(AdminUserInterface $adminUser): void
     {
         $this->client->buildUpdateRequest((string) $adminUser->getId());
     }
 
+    /**
+     * @When /^I (?:|upload|update) the "([^"]+)" image as (my) avatar$/
+     */
+    public function iUploadTheImageAsMyAvatar(string $avatar, AdminUserInterface $administrator): void
+    {
+        $response = $this->client->upload(
```

Comment:
Shouldn't we also use `avatarImageClient` for the avatar uploading?

Reply:
To be honest, there is no difference as I pass entire url here. Do you suggest to refactor upload method?

Reply:
It's an option, but for me, the main problem is confusion, that we upload an avatar with `$client`, but remove it with the `avatarImageClient` 🐎

Reply:
I would vote for doing it the same way with the rest of our resources. Right now upload function is behaving differently than other client methods, so +1 for refactor